internal/http/auth.go

```diff
@@ -44,3 +44,9 @@ type InsightsInsertKeyAuthorizer struct{}
 func (a *InsightsInsertKeyAuthorizer) AuthorizeRequest(r *Request, c *config.Config) {
 	r.SetHeader("X-Insert-Key", c.InsightsInsertKey)
 }
+
+type LicenseKeyAuthorizer struct{}
+
+func (a *LicenseKeyAuthorizer) AuthorizeRequest(r *Request, c *config.Config) {
+	r.SetHeader("X-License-Key", c.LicenseKey)
+}
```

Comment:
I'd be hesitant to add License key here.  IIRC Logs support sending via insert keys which already exist.

Reply:
Their preferred method is License Key and I had no luck getting an Insert Key to work

Reply:
😢

Reply:
I agree, seems like leaving sharp knives out.